Felienne/hedy
# [CLEAN-UP] Replaces mail YAMLs with gettext

**Description**
In this PR we move the mail translations from YAML files to the gettext structure, simplifying our codebase. This is mostly a clean-up task as we simply move the translations. But as we re-write some back-end structure as well some testing is needed that everything still works.

**Fixes**
This PR fixes #2476.

**How to test**
This cannot be tested locally. When the tests pass we should merge and verify that it's all working with the templates by performing the following actions in multiple languages:

- Create an account (receive a verification mail -> verify that link is working)
- Reset password
- Recover password
- Set account as teacher (only the admin can do this @Felienne) -> also receive mail
